mock-backend.js

```diff
@@ -52,9 +55,10 @@ function sortBurgersByPrice(burgers) {
 module.exports = {
     createRestaurant,
     getRestaurants,
-    createMenuItem,
+    createDish,
     getAllBurgers,
-    sortBurgersByPrice
+    sortBurgersByPrice,
+    runTests
```

Comment:
It doesn't look like there's any need to export the `runTests` function

Reply:
yup